app/assets/javascripts/discourse/tests/acceptance/admin-site-settings-test.js

```diff
@@ -1,6 +1,7 @@
+import { exists } from "discourse/tests/helpers/qunit-helpers";
 import { fillIn, click, visit, currentURL } from "@ember/test-helpers";
 import { test } from "qunit";
-import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import { acceptance, count } from "discourse/tests/helpers/qunit-helpers";
```

Comment:
(btw. at some point I'd like to ask prettier or eslint nicely to merge imports from the same file, and maybe sort them alphanumerically 😃)

Reply:
I was thinking the same thing. I think once I'm done this I will figure out a way to condense all the imports.